doc_source/private-clusters.md

```diff
@@ -69,4 +70,4 @@ The following [VPC endpoints](https://docs.aws.amazon.com/vpc/latest/userguide/v
 + `com.amazonaws.<region>.sts` – If using AWS Fargate or IAM roles for service accounts
```

Comment:
Need to add "or cluster autoscaler" to this line.

Reply:
Updated and did [commit](https://github.com/awsdocs/amazon-eks-user-guide/pull/374/commits/e93c03fc0f541726551d04b65591961a97301c2c) under this same pull request